packages/ember-routing/lib/services/router.js

```diff
@@ -197,9 +197,10 @@ const RouterService = Service.extend({
    */
   isActive(...args) {
     let { routeName, models, queryParams } = extractRouteArgs(args);
+    let router = this._router;
```

Comment:
Could do `let { _isActiveIntent, state } = this._router;` and use those instead.